vortexasdk/client.py

```diff
@@ -195,6 +197,23 @@ def create_client() -> VortexaClient:
         raise KeyError(
             "VORTEXA_API_KEY environment variable is required to use the VortexaSDK"
         )
+    latest_version, _ = get_latest_package_version(pkg_name)
+    try:
+        version_mask = StrictVersion(__version__) < StrictVersion(
+            latest_version
+        )
+    except ValueError as e:
+        if "invalid version number" in str(e):
+            version_mask = LooseVersion(__version__) < LooseVersion(
+                latest_version
+            )
+        else:
+            raise e
+    if version_mask:
+        logger.warning(
+            f"You are using vortexasdk version {__version__}, however version {latest_version} is available.\n"
+            f"You should consider upgrading via the 'pip install vortexasdk --upgrade' command."
+        )
```

Comment:
I added the warning at the point where we are creating the client. Not sure if that's the best place to do so. Also open to any comments on whether we could do this code more robust, i.e. that's a 'nice-to-have' check, we don't want this to be the cause of failing to create the client.